lib/src/platform/src/push_activation_events_internal.dart

```diff
@@ -2,10 +2,19 @@ import 'dart:async';
 
 import 'package:ably_flutter/ably_flutter.dart';
 
+/// package-private implementation of [PushActivationEvents]
+/// used internally by [Push] instances
+// FIXME: StreamControllers here may leak, we should add a `close()` method
```

Comment:
Maybe let's create a Github issue for it and add a link to the issue here? :thinking:

Reply:
Updated the comment with link to https://github.com/ably/ably-flutter/issues/382